app/models/section_edition.rb

```diff
@@ -39,9 +39,9 @@ class SectionEdition
 
   scope :with_slug_prefix, ->(slug) { where(slug: /^#{slug}.*/) }
 
-  index "section_id"
-  index "state"
-  index "updated_at"
+  index section_id: 1
+  index state: 1
+  index updated_at: 1
```

Comment:
Is it worth extracting these magic numbers into an intention-revealing named constant? That is assuming we know what the magic number means!